### PR TITLE
Implement poll API

### PR DESCRIPTION
**Pull request by setval**

### CODE REVIEW

**Comment by demget** [comment]
Thank you!

File: poll.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+func (p *Poll) IsRegular() bool {
+	return p.Type == "regular"
+}
+
+func (p *Poll) IsQuiz() bool {
+	return p.Type == "quiz"
+}
+
+func (p *Poll) AddAnswers(text ...string) {
```
Comment (stek29) [inline poll.go:43]: why is it named AddAnswers, if it actually adds Options?

Comment (setval) [reply poll.go:43]: Initially, it was planned that options (PollOption) only contains Text, so I decided to call the method this way (that is, the VoterCount field is not filled in). But now I really understand that perhaps another name would be acceptable. I reserve the right for collaborators.

Comment (stek29) [reply poll.go:43]: @demget maybe it’s a good idea to change it before develop is merged?

**Pull request merged**